docs/agents/magpie.rst

```diff
@@ -0,0 +1,121 @@
+.. highlight:: rst
+
+.. _magpie:
+
+===============
+Magpie Agent
+===============
+
+.. image:: ../_static/images/magpie.jpeg
+  :width: 400
+  :alt: Alternative text
+
+The magpie is an incredibly intelligent bird, with a decent ability to mimic
+other bird calls, though not as good as `the superb lyrebird
+<https://www.youtube.com/watch?v=mSB71jNq-yQ>`_. In the context of OCS, the job
+of the Magpie agent is to parse data being passed from the SMuRF Streamer, and
+to translate it into a stream that Lyrebird can interpret and display.
+
+The Magpie agent should be setup and run on the same node that is running
+lyrebird, which must have some sort of graphics card (either integrated or
+dedicated), and must be on the same network as the smurf-server for real-time
+playback.
```

Comment:
Why must the agent be on the node running lyrebird?  I suspect one can tunnel away this requirement... is there a reason someone should not do that?

I think it would be helpful to have a paragraph describing how you co-operate this with Lyrebird ... launch the agent, then start the main process, then launch lyrebird with default parameter?  Or do you need some particular lyrebird config file?